src/extension.ts

```diff
@@ -143,6 +147,17 @@ export async function activate(ctx: vscode.ExtensionContext) {
     }
 }
 
+function informUserAboutLspIfTf12() {
+    try {
+        const versionResponse = cp.execSync("terraform --version");
```

Comment:
probably better to use `Runner.getRequiredExecutable(VersionRequirement.parse(">= 0.9.1"))` that way you will check for all configured terraforms (as you can specify multiple executables via config)`

Reply:
Ooh interesting. I took a look and I'm a bit worried that if they don't have `tf 0.12` then this would thow an `indexOutOfRange` error due to the the `[0]` at the end of the line. Is that right?

```
 getRequiredExecutable(requirement: VersionRequirement): TerraformExecutable {
    const supported = this.executables.filter(e => e.version && requirement.isFulfilledBy(e.version));
    return supported.sort((left, right) => TerraformVersion.compare(left.version, right.version))[0];
  }
```